Development.md

```diff
@@ -116,6 +116,15 @@ i-146c2f97  on     active  2 hours ago  1   0.00%  3.21%
 i-c7de605c  on     active  2 hours ago  0   0.00%  0.00%
 ```
 
+## Build Image
+
+If you're working on the builder, you can set the [`BuildImage` Rack param](https://convox.com/docs/rack-parameters#buildimage) to a Docker image for the builder:
+
+```
+$ make builder
+$ convox rack params set BuildImage=convox/build:$(whoami)
```

Comment:
Only users of the Convox docker hub would be able to use `make builder` without changes. Could we be more explicit there?